spec/support/aws_helpers.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+module AwsHelpers
+  # rubocop:disable Metrics/MethodLength
```

Comment:
seems simpler to leave it all in one method, it's only stubbing like 4 lines of code